services/userService.js

```diff
@@ -0,0 +1,39 @@
+const { set, get } = require('./cacheService')
+const {
+  fetchUser
+} = require('../models/users')
+
+const cacheUser = (user) => {
+  if (!user) {
+    return false
+  }
+  const userObject = {
+    github_id: user.github_id,
+    id: user.id,
+    username: user.username
+  }
+  set(user.id, userObject)
```

Comment:
Can we store the user object into the cache?
In that we, we can also utilise the same cache to get the user data during auth and will reduce the response time.

It would be great if you can create an issue for this cache implementation(if not created) and document all stuff into it.